docs/cep/getting-started-stream-workers.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 10
+title: Getting Started with Stream Workers
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to create a stream worker that reads data added to a collection and sends the data to a stream.
+
+## Create a Stream Worker
+
+Refer to the [Stream Workers](/cep/index.md) section for an explanation of how stream workers function.
```

Comment:
Good addition, but I'm not sure this link is correct. Shouldn't it just be the filename, since they are in the same folder?

Reply:
```suggestion
Refer to the [Stream Workers](index.md) section for an explanation of how stream workers function.
```